EvidenceApi.Tests/V1/E2ETests/EvidenceRequestsTest.cs

```diff
@@ -151,18 +187,177 @@ public async Task ReturnNotFoundWhenCannotFindEvidenceRequest()
         {
             var fakeId = "ed0f2bd2-df90-4f01-b7f1-d30e402386d0";
             var uri = new Uri($"api/v1/evidence_requests/{fakeId}", UriKind.Relative);
-            var responseFind = await Client.GetAsync(uri).ConfigureAwait(true);
+            var responseFind = await Client.GetAsync(uri);
             responseFind.StatusCode.Should().Be(404);
         }
 
+        [Test]
+        public async Task CreateDocumentSubmissionUnsuccessfulWhenNoDocumentTypeProvided()
+        {
+            var entity = _fixture.Build<EvidenceRequest>()
+                .With(x => x.DocumentTypes, new List<string> { "passport-scan" })
+                .With(x => x.DeliveryMethods, new List<DeliveryMethod> { DeliveryMethod.Email })
+                .Without(x => x.Communications)
+                .Without(x => x.DocumentSubmissions)
+                .Create();
+            DatabaseContext.EvidenceRequests.Add(entity);
+            DatabaseContext.SaveChanges();
+
+            string body = "{}";
+            var jsonString = new StringContent(body, Encoding.UTF8, "application/json");
+            var uri = new Uri($"api/v1/evidence_requests/{entity.Id}/document_submissions", UriKind.Relative);
+            var response = await Client.PostAsync(uri, jsonString).ConfigureAwait(true);
```

Comment:
There's some left over `ConfigureAwait(true)` here.

Reply:
Ah thanks for spotting that! I'll remove it.